app/views/proposals/show.html.haml

```diff
@@ -75,31 +69,6 @@
     %li
       = link_to "View history", history_proposal_path(@proposal)
 
-- # @todo: there are multiple approval-status ids
-- if @proposal.parallel?
```

Comment:
this is what a parallel proposal looks like when there are no approvals and when all parallel steps are approved:

![screen shot 2015-12-16 at 10 05 21 am](https://cloud.githubusercontent.com/assets/601515/11849394/7d1ed6c6-a3dc-11e5-83c5-47fa6da72059.png)

When I update the status on the proposal itself to "approved", this is what it looks like:

![screen shot 2015-12-16 at 10 06 56 am](https://cloud.githubusercontent.com/assets/601515/11849404/899c3ec0-a3dc-11e5-8b71-b4b641553bb0.png)